woocommerce/woocommerce-admin
# Add Jetpack flow to onboarding profiler for release 1.1.1

Variant of https://github.com/woocommerce/woocommerce-admin/pull/4380 that uses the existing component instead of a new one.  Check https://github.com/woocommerce/woocommerce-admin/pull/4380 first before approving/merging this one.

### Detailed test instructions:

1. Make sure Jetpack is not activated.
1. Enable the onboarding profiler.
1. Walk through to the plugin benefits screen and click "Yes Please!"
1. You should be redirected to the Jetpack connection.
1. After approving the connection, you should be redirected back to the task list.
